examples/resources/xray_watch/resource.tf

```diff
@@ -67,6 +67,34 @@ resource "xray_watch" "repository" {
   watch_recipients = ["test@email.com", "test1@email.com"]
 }
 
+resource "xray_watch" "all-builds-with-filters" {
+  name        = "build-watch"
+  description = "Watch all builds with Ant patterns filter"
+  active      = true
+
+  watch_resource {
+    type       = "all-builds"
+    bin_mgr_id = "default"
+
+    ant_filter {
+      exclude_patterns = ["a*", "b*"]
+      include_patterns = ["ab*"]
+    }
+  }
+
+  assigned_policy {
```

Comment:
Side note, I feel like it might be useful to have `assigned_license_policy` and such like we do with RT